### PR TITLE
Logic submit claim

### PR DESCRIPTION
**Pull request by AdrianSalas1992**

### CODE REVIEW

**Review by MauricioJChavez — APPROVED** [review]
Changes look good

File: frontend/jce-app/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
-    "axios": "^1.5.1",
+    "axios": "^1.6.2",
```
Comment (DiegoE-prog) [inline frontend/jce-app/package.json]: Don't change the version.

File: frontend/jce-app/src/views/Claims/ClaimInformation.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { Time } from "phaser";
```
Comment (DiegoE-prog) [inline frontend/jce-app/src/views/Claims/ClaimInformation.js]: Remove this, with this we can't run the application

**Review by DiegoE-prog — CHANGES_REQUESTED** [review]
Please check the notes that I leave.

File: frontend/jce-app/src/views/Claims/ClaimInformation.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { Time } from "phaser";
+
 import React, {useState} from "react"
-import Axios from "axios";
+import xios from "axios";
```
Comment (DiegoE-prog) [inline frontend/jce-app/src/views/Claims/ClaimInformation.js]: This will not work

**Review by DiegoE-prog — CHANGES_REQUESTED** [review]

**Pull request merged**